templates/about.hbs

```diff
@@ -168,6 +168,9 @@ rustc-args = [ "--example-rustc-arg" ]
 # Additional `RUSTDOCFLAGS` to set (default: none)
 rustdoc-args = [ "--example-rustdoc-arg" ]</pre></code>
 
+  <h4>Test crate documentation build locally</h4>
+  <p>The <a href="https://github.com/rust-lang/docs.rs/blob/master/README.md#build-subcommand" target="_blank">docs.rs README</a> details how to build an unpublished crate's documentation locally using the same Docker image used by the build agent.</p>
```

Comment:
nit:
```suggestion
  <p>The <a href="https://github.com/rust-lang/docs.rs/blob/master/README.md#build-subcommand" target="_blank">docs.rs README</a> describes how to build an unpublished crate's documentation locally using the same Docker image used by the build agent.</p>
```

Reply:
Your wording is definitely an improvement.